PlaceRouteHierFlow/placer/SeqPair.cpp

```diff
@@ -1178,6 +1178,7 @@ vector<int> SeqPair::SwapTwoListinSeq(vector<int>& Alist, vector<int>& Blist, ve
       } else if ( (*ait)>(*bit) ) {
         newApos.push_back(*bit); ++bit;
       } else {
+        newApos.push_back(*bit); newApos.push_back(*bit); ++bit; ++ait;
```

Comment:
I am not sure why same index in two lists are ignore. So I still merge sort them. At least there is no infinite loop here.